serviceUsage.html

```diff
@@ -66,9 +66,24 @@ <h2>
         <pre>set JAVA_OPTS="-Djava.ext.dirs=/path/to/jars/" in catalina.sh</pre>
         <p><strong>NOTE:</strong> in the above example, protocol jar file must be present inside "/path/to/jars/" folder.</p>
         <p><strong>NOTE:</strong> <code>-Djava.ext.dirs</code> is no longer working in some JAVA8 versions and in JAVA9.
-            So users should create a wrapper project to include both Generate/Publish and their protocol in
+            So users should create a wrapper project to includse both Generate/Publish and their protocol in
             or place the protocol library in the folder for external dependencies of their JVM installation.
         </p>
+
```

Comment:
The section for running inside tomcat should get its own header and should be moved after the the section for running without tomcat.

Reply:
Fixed.